packages/@uppy/informer/src/index.js

```diff
@@ -24,49 +29,36 @@ module.exports = class Informer extends UIPlugin {
   }
 
   render = (state) => {
-    const { isHidden, message, details } = state.info
-
-    function displayErrorAlert () {
-      const errorMessage = `${message} \n\n ${details}`
-      alert(errorMessage)
-    }
-
-    const handleMouseOver = () => {
-      clearTimeout(this.uppy.infoTimeoutID)
-    }
-
-    const handleMouseLeave = () => {
-      this.uppy.infoTimeoutID = setTimeout(this.uppy.hideInfo, 2000)
-    }
-
     return (
-      <div
-        className="uppy uppy-Informer"
-        aria-hidden={isHidden}
-      >
-        <p role="alert">
-          {message}
-          {' '}
-          {details && (
-            <span
-              aria-label={details}
-              data-microtip-position="top-left"
-              data-microtip-size="medium"
-              role="tooltip"
-              onClick={displayErrorAlert}
-              onMouseOver={handleMouseOver}
-              onMouseLeave={handleMouseLeave}
-            >
-              ?
-            </span>
-          )}
-        </p>
+      <div className="uppy uppy-Informer">
+        <TransitionGroup>
+          {state.info.map((info) => (
+            <FadeIn key={info.message}>
+              <p role="alert">
+                {info.message}
+                {' '}
+                {info.details && (
+                  <span
+                    aria-label={info.details}
+                    data-microtip-position="top-left"
+                    data-microtip-size="medium"
+                    role="tooltip"
+                    // eslint-disable-next-line no-alert
+                    onClick={() => alert(`${info.message} \n\n ${info.details}`)}
```

Comment:
I'll siding with eslint on this one, I think we should stop using `alert` 😳 
I don't have an easy replacement solution in mind though, but I think we should at least add a TODO.

Reply:
Yes, I guess we should implement something custom, but this is only used for debugging, when someone clicks the error message, so not sure it’s too bad as is.

Reply:
If it's for debug-only, `console.log` should be fine, right?

Reply:
No, we want non-technical users to be able to report errors to us, when we ask for details for instance.